### PR TITLE
chore: upgrade `@vercel/og` lib

### PR DESCRIPTION
**Pull request by hbjORbj**
## What does this PR do?

- housekeeping: having important packages up to date

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A - I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: packages/ui/components/head-seo/HeadSeo.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ export const HeadSeo = (props: HeadSeoProps): JSX.Element => {
       ...seoObject.openGraph,
       images: [...(nextSeoProps.openGraph?.images || []), ...seoObject.openGraph.images],
     },
-    additionalMetaTags: [...(nextSeoProps.additionalMetaTags || [])],
```
Comment (hbjORbj) [inline packages/ui/components/head-seo/HeadSeo.tsx:134]: `seoObject.additionalMetaTags` wasn't being used

File: apps/web/pages/api/social/og/image.tsx [hunk 2]
```diff
@@ -83,7 +83,7 @@ export default async function handler(req: NextApiRequest) {
           />
         ),
         ogConfig
-      ) as { body: Buffer };
```
Comment (hbjORbj) [inline apps/web/pages/api/social/og/image.tsx:86]: Upgrading the lib makes casting not needed anymore

**Review by zomars — APPROVED** [review]
LGTM

**Comment by zomars** [comment]
### Merge activity

* **Nov 13, 3:50 PM EST**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/calcom/cal.com/17503).

**Pull request merged**